wireshark/USAGE.md

```diff
@@ -1,43 +1,51 @@
-# OQS-wireshark
+This project provides a Docker image to build [Wireshark](https://www.wireshark.org/) with quantum-safe cryptography
+support through the [Open Quantum Safe (OQS) provider](https://github.com/open-quantum-safe/oqs-provider). This Docker
+image allows Wireshark to analyze network traffic encrypted with post-quantum cryptographic protocols.
 
-This docker image contains a version of [wireshark](https://www.wireshark.org/) built to also properly display quantum-safe crypto (QSC) TLS operations.
+## Quick Start Guide
 
-To this end, it contains references to algorithms supported by [liboqs](https://github.com/open-quantum-safe/liboqs) and [OQS-OpenSSL](https://github.com/open-quantum-safe/openssl) from the [OpenQuantumSafe](https://openquantumsafe.org) project.
+```bash
+git clone https://github.com/open-quantum-safe/oqs-demos
+cd oqs-demos/wireshark
+docker build -t wireshark-oqs .
+docker run --rm -it --net=host -e DISPLAY=<your_host_ip>:<your_display_port> -v /tmp/.X11-unix:/tmp/.X11-unix wireshark-oqs
+```
 
-The image is based on Ubuntu and requires the host to run the Unix X-Window system.
+Replace `<your_host_ip>` with your IP address (e.g., `192.168.x.x`) and `<your_display_port>` with your display port,
+typically `:0`.
 
-## Quick start
+## Running Wireshark
 
-Execute this command to open the wireshark window on your host:
+You can run the Wireshark Docker container on Linux, Windows, or macOS using the following command:
```

Comment:
As discussed elsewhere, is this statement really correct? Has this been tested OK under these platforms? The reference to ".X11-unix" seems only to apply to X-Windows based systems, not to Windows or MacOS...

Reply:
In that case, I'd suggest removing this statement as it may mislead users. Maybe put a statement like "MacOS not tested. Feedback/suggestions for improvement for that platform welcome at https://github.com/open-quantum-safe/oqs-demos/issues"?